contracts/staking/src/contract.rs

```diff
@@ -205,6 +205,7 @@ pub fn execute(
         ExecuteMsg::CircuitBreaker {} => circuit_breaker(deps, env, info),
         ExecuteMsg::ResumeContract {} => resume_contract(deps, env, info),
         ExecuteMsg::RecoverPendingIbcTransfers {} => recover(deps, env, info),
+        ExecuteMsg::FeeWithdraw { amount } => fee_withdraw(deps, env, info, amount),
```

Comment:
This question might be because i haven't tracked the code for the past weeks. How are we going to know the accumulated fee amount since it will be in the contract address?